presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotQueryGenerator.java

```diff
@@ -85,6 +85,7 @@
                     .put("sum", "sum")
                     .put("distinctcount", "DISTINCTCOUNT")
```

Comment:
I didn't follow the need for adding a distinct count function, when apparently that one already is present above on L85 ? It seems to me that only the segmentpartitioneddistinctcount is the new one here ?

Reply:
The purpose of this is to allow user be able to run same query in pinot as well through presto pinot connector,
and users don't need to find the right translation for this(although some of them are pretty trivial).